posts/lorentz-center-meeting.md

```diff
@@ -0,0 +1,62 @@
+---
+author: 'Jana Göken'
+category: 'overview'
+date: 2023-08-01 06:30:08 UTC+02:00
+description: 'Explore my journey through the Machine-Checked Mathematics workshop, where I delve into the world of proof assistants and the vibrant Lean community.'
+has_math: true
+link: ''
+slug: lorentz-center-meeting
+tags: 'Machine-Checked_Mathematics, Workshop_Experiences, Lorentz_Center, Inclusive_Environment'
+title: My Experience at the Machine-Checked Mathematics workshop
+type: text
+---
+# My Experience at the Machine-Checked Mathematics workshop
+Hello, everyone! My name is Jana Göken, a master's student in mathematics from Bremen. Today, I want to share with you my experiences at the Machine-Checked Mathematics workshop that introduced me to the world of proof assistants, specifically Lean. The workshop took place in Leiden from July 10th to July 14th 2023, and it was an amazing and educational journey.
+
+<!-- TEASER_END -->
+# My First Encounter with Lean
+The first time I heard about the proof assistant Lean, or any proof assistant, was in the summer of 2022 when I attended the International Congress of Mathematicians for Number Theory and Algebraic Geometry in Zürich. There, I took the opportunity to attend a fascinating talk by Kevin Buzzard about the proof assistant Lean. I was instantly intrigued because, with the current rise of AI, I saw the great potential in the idea of using Lean to aid in mathematical proofs. Despite the excitement, I wasn't sure whether I would be able to start my own journey with Lean by myself.
```

Comment:
```suggestion
The first time I heard about the proof assistant Lean, or any proof assistant, was in the summer of 2022 when I attended the International Congress of Mathematicians for Number Theory and Algebraic Geometry in Zürich. There, I took the opportunity to attend [a fascinating talk by Kevin Buzzard](https://www.youtube.com/watch?v=SEID4XYFN7o) about the proof assistant Lean.
I was instantly intrigued because, with the current rise of AI, I saw the great potential in the idea of using Lean to aid in mathematical proofs. Despite the excitement, I wasn't sure whether I would be able to start my own journey with Lean by myself.
```